content/linux-gaming.md

```diff
@@ -60,6 +61,45 @@ Steam -> Settings -> Steam Play -> Check the box for "Enable Steam Play for supp
 
 ![Steam Settings](/images/linux-gaming/enable-steam-play_proton.png)
 
+## Using Steam
+
+Once installed, use the Application button to search for and run <u>Steam</u>.
```

Comment:
The button says "Applications", so I'm thinking Application should probably be plural to match the button, right?